packages/example-getting-started/docs/8-putting-it-together.md

```diff
@@ -0,0 +1,54 @@
+### Putting it all together
+
+Now that we've got all of our artifacts made, let's set them up in our
+application!
+
+We'll define a new helper function for setting up the repositories, as well
+as adding in our new controller binding.
+
+#### src/application.ts
+```ts
+import {ApplicationConfig} from '@loopback/core';
+import {RestApplication} from '@loopback/rest';
+import {TodoController, PingController} from './controllers';
+import {
+  Class,
+  Repository,
+  RepositoryMixin,
+  DataSourceConstructor,
+} from '@loopback/repository';
+import {db} from './datasources/db.datasource';
+import {TodoRepository} from './repositories';
+
+export class TodoApplication extends RepositoryMixin(RestApplication) {
+  constructor(options?: ApplicationConfig) {
+    super(options);
+    this.setupControllers();
+    this.setupRepositories();
+  }
+
+  setupControllers() {
+    this.controller(TodoController);
+    this.controller(PingController);
+  }
+
+  setupRepositories() {
+    // This will allow you to test your application without needing to
+    // use the "real" datasource!
+    const datasource =
+      this.options && this.options.datasource
+        ? new DataSourceConstructor(this.options.datasource)
+        : db;
+    this.bind('datasource').to(datasource);
+    this.repository(TodoRepository);
+  }
+}
+```
+
+### Try it out
+
+Now that your app is ready to go, try it out with your favourite REST client!
+Start the app (`npm start`) and then make some REST requests:
+- `POST /todo` with a body of `{ "title": "get the milk" }`
```

Comment:
We probably should illustrate the usage of `API Explorer` and/or `curl` here.

Reply:
I don't intend to improve the individual documents at this time. There's quite a bit of work required across all of the pieces to make it a more cohesive and instructive tutorial, but I want to make sure that the wireup between this repository and loopback.io is finished first.

Reply:
That's fair. I'm fine with follow-up PRs.